webportal/package.json

```diff
@@ -63,7 +63,8 @@
     "uglifyjs-webpack-plugin": "1.1.4",
     "util": "0.10.3",
     "webpack": "3.10.0",
-    "winston": "2.4.0"
+    "winston": "2.4.0",
+    "yarn": "1.5.1"
```

Comment:
why do you add `yarn` here?

Reply:
Once I noticed that I didn't have yarn installed (so I couldn't run `yarn install`) on my machine and then used `npm install yarn` to install it. Then there came this line. Is this way of installing yarn correct?

Reply:
For windows, [msi installer](https://yarnpkg.com/en/docs/install) can be used; for linux, `yarn` is installed by `apt-get` and included by default, like node docker image or node js travis ci environment.
Since `yarn install` reads `package.json`, I think including it here is not a good idea. We can assume developers have prepared node js development environment (node, npm, yarn) already.